### ISSUE TITLE
nicehash miner not launching?

### ISSUE THREAD

**Issue by Mcrinaeft** [nicehash/NiceHashMiner-Archived#1475]
So, I downloaded nicehash miner and I want to try it out, But, I can't use it because it won't launch, It's running in the background and doesn't appear on the taskbar, I tried to uninstall it an reinstall it but it still didn't work, My laptop is an acer aspire vn7 v792g if that helps. How to fix? Also, It's windows 10 without any antiviruses because I deleted them thinking it'll fix the problem.

**Comment by martojrz** [comment]
Hi mcrinaeft, i have the same issues right now
did u already solve that issue?

**Comment by Mcrinaeft** [comment]
Nope, I've given up all hope on nicehash miner, I tried literary everything to fix it but nothing worked, Sorry for not being helpful.
